Cargo.lock

```diff
@@ -13,9 +13,9 @@ dependencies = [
 
 [[package]]
 name = "bindgen"
-version = "0.69.5"
+version = "0.69.4"
```

Comment:
Yes, great, thanks, this is what I meant. For the record, here is mozilla-central's `Cargo.lock` using `bindgen` `0.69.4`:

https://searchfox.org/mozilla-central/rev/4bacdbc8ac088f2ee516daf42c535fab2bc24a04/Cargo.lock#450-452